test/configs/syntaxerr.py

```diff
@@ -21,4 +21,4 @@
 # It is really dumb that we need a giant copyright notice for this one line
 # file. Fucking lawyers :(
 
-sdf [
```

Comment:
You should make this `sdf [  # noqa`, we want to test something that is incorrect